backgroundJobs/Addresses.js

```diff
@@ -39,14 +39,19 @@ const oneInchOracleAddress = {
     "XDAI" : "0x142DB045195CEcaBe415161e1dF1CF0337A4d02E"
 }
 
-//TODO: add "NEAR"
-const multicallAddress = { "ETH" : "0x5BA1e12693Dc8F9c48aAD8770482f4739bEeD696", "BSC" : "0xcb6E2F66DF0493b4dD23ac7727D1677D1208b697",
-    "AVAX" : "0xcb6E2F66DF0493b4dD23ac7727D1677D1208b697",
-    "MATIC" : "0xe539B93620570A92EF90eF7E60c79d9fAee91186", "NEAR": "0xcb6E2F66DF0493b4dD23ac7727D1677D1208b697",
-    "FTM" : "0xab35D115974aC0A3C4Bd16A70DF77003c9f4C011",
-    "ARBITRUM" : "0x0c05e6968AED7Ca120464a78731144e75052CeB1",
-    "CRO" : "0x5e954f5972EC6BFc7dECd75779F10d848230345F",
-    "MOONBEAM" : "0x6477204E12A7236b9619385ea453F370aD897bb2" }
+const multicallAddress = {
+	"ETH": "0x5BA1e12693Dc8F9c48aAD8770482f4739bEeD696",
+	"BSC": "0xcb6E2F66DF0493b4dD23ac7727D1677D1208b697",
+	"AVAX": "0xcb6E2F66DF0493b4dD23ac7727D1677D1208b697",
+	"MATIC": "0xe539B93620570A92EF90eF7E60c79d9fAee91186",
+	"NEAR": "0xcb6E2F66DF0493b4dD23ac7727D1677D1208b697",
+	"FTM": "0xab35D115974aC0A3C4Bd16A70DF77003c9f4C011",
+	"ARBITRUM": "0x0c05e6968AED7Ca120464a78731144e75052CeB1",
+	"CRO": "0x5e954f5972EC6BFc7dECd75779F10d848230345F",
+	"MOONBEAM": "0x6477204E12A7236b9619385ea453F370aD897bb2",
+    "OPTIMISM": "0xcA11bde05977b3631167028862bE2a173976CA11"
```

Comment:
fix indentation

Reply:
Indentation is good locally, github probably display it wrong